code/modules/client/preferences_savefile.dm

```diff
@@ -104,6 +104,9 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	if (current_version < 43)
 		migrate_legacy_sound_toggles(savefile)
 
+	if (current_version < 45) // Feel free to bump this version up if more quirks are added to this proc
+		migrate_quirks_to_loadout(save_data)
```

Comment:
Not the right solution to this, too easy to just update migrate_quirks_to_loadout and nothing else.

I think instead this should be something like `migrate_quirk_to_loadout("Pride Pin", whatever)`, and then we write them out manually here since I can't imagine we ever do more than a handful at a time